Baskerville/llama-download.md

```diff
@@ -0,0 +1,125 @@
+# Llama download on Baskerville
+
+We’ll be downloading the models from:
+
+https://llama.meta.com/llama-downloads
+
+This involves agreeing to the T&Cs in order to get keys, then following the instructions in the [REAMDE](https://github.com/meta-llama/llama3).
```

Comment:
```suggestion
This involves agreeing to the T&Cs in order to get keys, then following the instructions in the [README](https://github.com/meta-llama/llama3).
```

Reply:
Impressively spotted (had to read the suggested change with both lines next to each other several times before I got what was actually happening here!).